requests-for-comments/2023-11-28-slack-removal.md

```diff
@@ -0,0 +1,31 @@
+# Request for comments: Closing the PostHog Users Slack
+
+This RFC is to explore the topic of potentially closing the PostHog Users Slack entirely. 
+
+## Argument: Why would we close it?
+
+The PostHog Users Slack currently exists in limbo. It was previously the main channel for getting customer feedback, chatting to users, fostering a community, and running community support. Now, however, we have better and more scalable methods for each of these - Surveys, user interviews, the Post/Questions section, and the support modal respectively. 
+
+As we've introduced each of these tools, we've moved further from the Slack to the point that the user Slack is no longer referenced or linked to anywhere on the site, and team members admit they rarely check it. 
+
+The same isn't true for users, a small ([but declining](/images/slack.png)) number of whom still use it almost universally for submitting support requests. Often these don't get a response from us (because it's not a support channel) or other users (because we've failed to foster a true community on Slack).
+
+The result is that the Slack gives users a bad experience, while providing little to no benefit to us or them. There's no clear owner to make any changes, and so the argument is that closing will improve the user experience with little to no impact for the team. 
+
+## Counterarguments: Why would we keep it?
+
+Previously when I've raised this topic the argument has been that some teams use the Slack when dealing with users. Typically the argument has been that Customer Success can nurture clients through the Slack, that Engineering teams can get feedback and discuss contributions, and that Product/Marketing can use it for speaking with users directly. 
+
+> I no longer thing this is the case, and that these conversations happen in other channels. Please comment if you feel differently, or have other arguments. 
+
+A speculative argument to keep the Slack would be that we are planning to invest more in the PostHog community, and that we want to build a genuine network of users that are helping each other and sharing content. Currently Slack doesn't feature into those plans, however. 
+
+## How would we close it?
+
+If we were to close the Slack I'd create a more detailed action plan at the time, but essential steps would be: 
+
+1. Announce publicly that we're closing the Slack, why, and when. Send an email to all active members from the last month. We would offer all active users a reward as thanks for feedback - say, £15 credit for the PostHog store. 
```

Comment:
@corywatilo One thought I had was that we could try to migrate users and create a Community profile for them automatically, perhaps with a unique profile image or border. 

I've omitted that because I think it's a big ask, but food for thought!

Reply:
Can we throw Slack under the bus in our explanation?

- Member limit
- Not economical to pay per user
- Message history disappears, which makes it impossible to use as a knowledge repo

But...

Looking through recent Slack messages, it's great to see users trying to help each other out (eg: sharing links to questions on the website).

I don't think a blanket $15 coupon is necessary, but maybe we can parlay the goodwill of those who _are_ helping out to say, "Hey, come over over our community forums and help there, and you'll get freebies for answering q's!"

We, of course, need to build this, but it could be a good impetus to getting this instrumented.

cc @smallbrownbike

Reply:
+1 for use of 'parlay'

Reply:
I won't lie, I googled it.

Reply:
Ha, how long would you imagine to build something like that in terms of other priorities though? 

Not trying to push it up or down on your list, but I don't think it's worth introducing as a blocker. If we're going to close the Slack we should aim to do it as soon as possible, while there are still people to migrate and to avoid poor experiences. A $15 voucher (or other incentive) could accomplish much of the same thing and wouldn't mean waiting (presumed) weeks for us to build a new system. 

Agree on the other points though. We'd definitely mention the cost and index support in any full reasoning.

Reply:
1) The get money for answering questions in community forum is tricky, if someone provides a bad answer, what do we do? So I'd rather we don't build that.
2) Please 🙏  let's highlight the support button in the app for users to use for bug reports, not community forum, that's a much better experience for our support engineers.

Reply:
What about opening [GitHub Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/collaborating-with-maintainers-using-discussions)? Does anyone have experience with that? It's close to the code, and it could provide the "users help each other out" part when it's not about bugs and other issues.
It would also provide a way to actually move issues there that are not bugs and such. I always prefer to read "this got moved to discussion #xyz" instead of "this is not a bug, please ask in our forums".

Reply:
Personally, I think we should try to consolidate the community in one place - ideally a place we have full control over. We've invested in building a community platform on Posthog.com through Questions and Posts, with plans to extend that further. I feel at this point we should go all-in on that, rather than trying to divide discussion between two places. 

And, of course, if people want to make GitHub issues or contribute to the code, they can still do that.

Reply:
Today I searched for "posthog stop autocapture", and landed on a community [question](https://posthog.com/questions/how-do-you-disable-autocapture-site-wide) that had an answer that Alex gave some time ago. This would have never happened if the question was asked in Slack. Likely this question alone has saved a non-zero amount of support load.